README.md

```diff
@@ -324,6 +324,7 @@ PushNotification.localNotification({
   ignoreInForeground: false, // (optional) if true, the notification will not be visible when the app is in the foreground (useful for parity with how iOS notifications appear). should be used in combine with `com.dieam.reactnativepushnotification.notification_foreground` setting
   shortcutId: "shortcut-id", // (optional) If this notification is duplicative of a Launcher shortcut, sets the id of the shortcut, in case the Launcher wants to hide the shortcut, default undefined
   onlyAlertOnce: false, // (optional) alert will open only once with sound and notify, default: false
+  repeatTime: 1, // (optional) Increment of configured repeateType. Check 'Repeating Notifications' section for more info.
```

Comment:
Hi @johgusta 
This line should not be documented there, it's on the scheduldedNotification method 😉

Reply:
I might have misunderstood this, but I think repeating is possible for both the `localNotification` and `localNotificationSchedule` functions. Since this one has the documentation for `repeatType: 'day'` I thought this would be the right place to add this info. In my project we only use `localNotificationSchedule` though but as far as I can tell it should still work for both.